app/stylesheet/miq-structured-list.scss

```
@@ -1,12 +1,140 @@
 .miq-structured-list {
-  border: 1px solid lightgray;
   margin-bottom: 2rem !important;
 
+  .list_cell {
+    padding: 5px 0 5px 0;
+  }
+
+  .clickable {
+    cursor: pointer;
+
+    &:hover {
+      background: #f4f4f4;
+    }
+  }
+
+  .list_header {
+    .sortable_header {
+      display: flex;
+    }
+
+    &.bx--structured-list-th {
+      padding: 0.8em 0.5rem 0.5rem 0.3rem !important;
+
+      &:first-child {
+        padding: 0.8em 0.5rem 0.5rem 0.5rem !important;
+      }
+    }
+  }
+
+  .bx--structured-list-row {
+    border: 0;
+  }
+
   .bx--structured-list-td {
-    padding: 0.875rem !important;
+    padding: 0.3rem !important;
   }
 
   .label_header {
     font-weight: bold;
+    min-width: 20%;
+    width: 250px;
+    padding-left: 0 !important;
+  }
+
+  .content_value {
+    .content {
+      display: flex;
+      flex-direction: row;
+    }
+
+    .cell_link {
+      display: flex;
+      color: #0f62fe;
+    }
+
+    .multi_row_cell {
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+
+      .sub_row_item {
+        width: 100%;
+        display: flex;
+        padding: 5px 0 5px 0;
+
+        .sub_label {
+          width: 50%;
+          font-weight: bold;
```

@jeffibm Use number for font-weight. 
```suggestion
          font-weight: 600;
```